src/tests/unit/tests/DetailsView/components/save-assessment-button.test.tsx

```diff
@@ -36,71 +44,95 @@ describe('SaveAssessmentButton', () => {
             href: 'url',
             userConfigurationStoreData,
         };
-        eventStub = new EventStubFactory().createMouseClickEvent();
     });
 
     describe('on dialog enabled', () => {
-        let wrapper: ShallowWrapper;
+        describe('render', () => {
+            beforeEach(() => {
+                wrapper = render(<SaveAssessmentButton {...propsStub} />);
+                fireEvent.click(wrapper.getByRole('link'));
+            });
+            it('snapshot of dialog', () => {
+                wrapper = render(<SaveAssessmentButton {...propsStub} />);
 
-        beforeEach(() => {
-            wrapper = shallow(<SaveAssessmentButton {...propsStub} />);
-            wrapper.find(InsightsCommandButton).simulate('click', eventStub);
-        });
+                const dialog = wrapper.container.querySelector('mock-Dialog');
+                expect(dialog).toMatchSnapshot();
```

Comment:
Since the button is mocked, we should include it in the snapshot.
```suggestion
                expect(wrapper.asFragment()).toMatchSnapshot();
```